__tests__/ExpensiMark-HTML-test.js

```diff
@@ -609,6 +609,111 @@ test('Test a url ending with a closing parentheses autolinks correctly', () => {
+        {
+            testString: 'staging.new.expensify.com/details',
+            resultString: '<a href="https://staging.new.expensify.com/details" target="_blank" rel="noreferrer noopener">staging.new.expensify.com/details</a>',
+        },
+        {
+            testString: 'https://www.expensify.com?name=test&email=test@expensify.com',
+            resultString: '<a href="https://www.expensify.com?name=test&amp;email=test@expensify.com" target="_blank" rel="noreferrer noopener">https://www.expensify.com?name=test&amp;email=test@expensify.com</a>',
+        },
+        {
+            testString: 'https://staging.new.expensify.com/details?login=testing@gmail.com',
+            resultString: '<a href="https://staging.new.expensify.com/details?login=testing@gmail.com" target="_blank" rel="noreferrer noopener">https://staging.new.expensify.com/details?login=testing@gmail.com</a>',
+        },
+        {
+            testString: 'staging.new.expensify.com/details?login=testing@gmail.com',
+            resultString: '<a href="https://staging.new.expensify.com/details?login=testing@gmail.com" target="_blank" rel="noreferrer noopener">staging.new.expensify.com/details?login=testing@gmail.com</a>',
+        },
+        {
+            testString: 'http://necolas.github.io/react-native-web/docs/?path=/docs/components-pressable--disabled',
+            resultString: '<a href="http://necolas.github.io/react-native-web/docs/?path=/docs/components-pressable--disabled" target="_blank" rel="noreferrer noopener">http://necolas.github.io/react-native-web/docs/?path=/docs/components-pressable--disabled</a>',
+        },
+        {
+            testString: '-https://www.expensify.com /https://www.expensify.com @https://www.expensify.com',
+            resultString: '-<a href="https://www.expensify.com" target="_blank" rel="noreferrer noopener">https://www.expensify.com</a> /<a href="https://www.expensify.com" target="_blank" rel="noreferrer noopener">https://www.expensify.com</a> @https://www.expensify.com',
+        },
+        {
+            testString: 'expensify.com -expensify.com @expensify.com',
+            resultString: '<a href="https://expensify.com" target="_blank" rel="noreferrer noopener">expensify.com</a> -<a href="https://expensify.com" target="_blank" rel="noreferrer noopener">expensify.com</a> @expensify.com',
+        },
+        {
+            testString: 'https//www.expensify.com',
+            resultString: 'https//<a href="https://www.expensify.com" target="_blank" rel="noreferrer noopener">www.expensify.com</a>',
+        },
+        {
+            testString: '//www.expensify.com?name=test&email=test@expensify.com',
+            resultString: '//<a href="https://www.expensify.com?name=test&amp;email=test@expensify.com" target="_blank" rel="noreferrer noopener">www.expensify.com?name=test&amp;email=test@expensify.com</a>',
+        },
+        {
+            testString: '//staging.new.expensify.com/details?login=testing@gmail.com',
```

Comment:
we should render `://staging.new.expensify.com/details?login=testing@gmail.com` as link ? 
`://` will not be rendered. it's defined from old PR. I think we can use current test case as it is

Reply:
You mean that we need the following test case instead of with `//` ?
```
{
            testString: '://staging.new.expensify.com/details?login=testing@gmail.com',
            resultString: '://<a href="https://staging.new.expensify.com/details?login=testing@gmail.com" target="_blank" rel="noreferrer noopener">staging.new.expensify.com/details?login=testing@gmail.com</a>',
        },
```

Reply:
Maybe this is my old school thinking, but on browsers, a link like `://expensify.com` translates into `https://expensify.com` if the protocol being used on that page is https, and to `http://expensify.com` is the protocol is http. That's what made me think we should parse `://expensify.com` as a URL, but maybe we don't need to do that, just wanted to get more thoughts.

This isn't a blocker